common-develocity-gradle-configuration-groovy/settings.gradle

```diff
@@ -21,18 +21,18 @@ buildCache {
         enabled = true
     }
 
-    // Use the Gradle Enterprise connector's access key based authentication.
-    // This is available in Gradle Enterprise 2022.3+ and Gradle Enterprise Plugin 3.11+.
+    // Use the Develocity connector's access key based authentication.
+    // This is available in Develocity 2022.3+ and Develocity Plugin 3.11+.
     remote(gradleEnterprise.buildCache) {
         enabled = true
         push = isCI
     }
 
     // Use Gradle's built-in access credentials.
-    // This is available in all Gradle Enterprise and Gradle Enterprise Plugin versions.
+    // This is available in all Develocity and Develocity Plugin versions.
     /**
     remote(HttpBuildCache) {
-        url = 'https://enterprise-samples.gradle.com/cache/' // adjust to your build cache node, and note the trailing slash
+        url = 'https://develocity-samples.gradle.com/cache/' // adjust to your build cache node, and note the trailing slash
         allowUntrustedServer = false // ensure a trusted certificate is configured
         credentials { creds ->
             creds.username = System.getenv('GRADLE_ENTERPRISE_CACHE_USERNAME')
```

Comment:
Shoudln't we rename to `DEVELOCITY_CACHE_USERNAME` / `DEVELOCITY_CACHE_PASSWORD` ?

Reply:
This is a good point. i was thinking of also removing this section entirely since it is deprecated but I'll leave that to a separate PR. Created this to discuss: https://github.com/gradle/ge/issues/31447

Reply:
I'll rename in this PR.